requirements.txt

```
@@ -1,4 +1,4 @@
-certifi==2018.11.29
+certifi
```

Because otherwise it will be happy with any version of the package

```suggestion
certifi==2023.7.22
```

---

It doesn't just use the latest?
The build happens in a new Docker image each time so it won't find an existing package already installed?

---

I could probably change the install command to use the `--upgrade` parameter as well.
That would keep it on the latest I think?
```
 RUN pip3 install --upgrade -r requirements.txt --target /opt/app/python_modules
```

---

Okay I can see it does install the latest version, I guess it's okay